### PR TITLE
NUI-492 Support Windows 10+

### PR DESCRIPTION
**Pull request by DrGirlfriend**
[NUI-492 Support Windows 10+](https://jira.corp.adobe.com/browse/NUI-492)

I could not get a relative path to mocha working for windows hence the full path
Updated eslint and fixed complaints
Removed NodeJs test versions 10.15 & 10.19
Fixed a typo in a comment

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,13 +16,12 @@ os:
 
 node_js:
   - "10"
-  - "10.15"
-  - "10.19"
```
Comment (alexkli) [inline .travis.yml:20]: I think this library is used in both cli and actions, so we need to keep `10.15` and `10.19`.

File: package.json [hunk 2]
```diff
@@ -27,7 +28,19 @@
...
-    "test": "nyc --reporter=html --reporter=text ./node_modules/.bin/mocha ./test --recursive --exit",
-    "posttest": "eslint ./"
+    "test": "cross-os test-cross-os",
+    "posttest": "cross-os posttest-cross-os"
+  },
+  "cross-os": {
+    "test-cross-os": {
+      "darwin": "nyc --reporter=html --reporter=text ./node_modules/.bin/mocha ./test --recursive --exit",
+      "linux": "nyc --reporter=html --reporter=text ./node_modules/.bin/mocha ./test --recursive --exit",
+      "win32": "nyc --reporter=html --reporter=text C:\\Users\\travis\\build\\adobe\\node-fetch-retry\\node_modules\\.bin\\mocha.cmd --recursive --exit"
```
[the remark this answers is not in the document]
Comment (DrGirlfriend) [reply package.json:38]: ok but not for windows, it errors out saying mocha isn't a command unless the full path is used

File: package.json [hunk 3]
```diff
@@ -27,7 +28,19 @@
...
+    },
+    "posttest-cross-os": {
+      "darwin": "eslint ./",
+      "linux": "eslint ./",
+      "win32": "eslint ..\\"
```
[the remark this answers is not in the document]
Comment (DrGirlfriend) [reply package.json]: Windows doesn't understand `.` and I got mixed up somewhere and thought I tried no path for windows resulting in an error

[the remark this answers is not in the document]
Comment (DrGirlfriend) [reply package.json]: I got mixed up, I did try `.\\` but you are correct, `.` is just fine for everyone

**Pull request merged**